### PR TITLE
[Webstaurant_Store] Simplify ruleset.

### PR DESCRIPTION
**Pull request by bardiharborow**
The http redirect issues seem to have been resolved.

### CODE REVIEW

File: src/chrome/content/rules/Webstaurant_Store.xml [hunk 1]
```diff
@@ -1,18 +1,21 @@
...
-Fetch error: http://webstaurantstore.com/ => https://webstaurantstore.com/: Cycle detected - URL already encountered: https://www.webstaurantstore.com/
-Fetch error: http://www.webstaurantstore.com/ => https://www.webstaurantstore.com/: Cycle detected - URL already encountered: https://www.webstaurantstore.com/
-	Some pages redirect to http.
+
+	Problematic hosts in *webstaurantstore.com:
+
+		- cdnimg (different content)
+		- cdnimg1 (different content)
+		- cdnimg2 (different content)
+		- cdnimg3 (different content)
```
Comment (gloomy-ghost) [inline src/chrome/content/rules/Webstaurant_Store.xml]: These domains look good to me, such as https://cdnimg1.webstaurantstore.com/uploads/design/2014/1/webstaurantstore-logo.png
Can you give some links if there are any issues?

File: src/chrome/content/rules/Webstaurant_Store.xml [hunk 2]
```diff
@@ -1,18 +1,21 @@
...
 
 -->
-<ruleset name="The Webstaurant Store (partial)">
+<ruleset name="The Webstaurant Store">
 
 	<target host="webstaurantstore.com" />
+	<target host="cdn.webstaurantstore.com" />
+	<target host="chat.webstaurantstore.com" />
 	<target host="www.webstaurantstore.com" />
```
Comment (gloomy-ghost) [inline src/chrome/content/rules/Webstaurant_Store.xml]: `www.webstaurantstore.com` should come right after `webstaurantstore.com`

File: src/chrome/content/rules/Webstaurant_Store.xml (hunk 1 above)
Comment (bardiharborow) [reply src/chrome/content/rules/Webstaurant_Store.xml]: The implicit test URL gives 403 over https and 301 over http, and I'm not sure how to disable it.

File: src/chrome/content/rules/Webstaurant_Store.xml (hunk 2 above)
Comment (bardiharborow) [reply src/chrome/content/rules/Webstaurant_Store.xml]: Oh okay, I thought the sort order was alphabetical...

File: src/chrome/content/rules/Webstaurant_Store.xml (hunk 1 above)
Comment (gloomy-ghost) [reply src/chrome/content/rules/Webstaurant_Store.xml]: You can add an exclusion for the implicit URLs, try 
`<exclusion pattern="^http://cdnimg\d?\.webstaurantstore\.com/$"/>`

File: src/chrome/content/rules/Webstaurant_Store.xml (hunk 2 above)
Comment (gloomy-ghost) [reply src/chrome/content/rules/Webstaurant_Store.xml]: `www` is an exception for the alphabetical order as it is very common

**Comment by bardiharborow** [comment]
@gloomy-ghost done.

**Pull request merged**